src/helpers/util.js

```diff
@@ -39,5 +39,40 @@ module.exports = {
       ret.push(row)
     }
     return ret
+  },
+
+  /*
+    Binary Search:
+    Assumes that @arg array is sorted increasingly
+    returns the smallest i such that target >= changes[i]
+    returns arary.length - 1 (if all elements in array are smaller than target)
+    returns 0 (if target is smaller than the first element of array || if array is empty)
+  */
+  findLowerBound: function (target, array) {
+    if (array.length === 0) {
+      return 0
+    }
+    return findLowerBoundInternal(target, array, 0, array.length - 1)
+  }
+}
+
+function findLowerBoundInternal (target, array, lowerbound, higherbound) {
```

Comment:
If you use `length` instead of `higherbound`, it is obvious what is included and what is not.